packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -93,14 +94,30 @@ export class MirrorNodeClient {
     protected createAxiosClient(
         baseUrl: string
     ): AxiosInstance {
-        return Axios.create({
+        const axiosClient: AxiosInstance = Axios.create({
             baseURL: baseUrl,
             responseType: 'json' as const,
             headers: {
                 'Content-Type': 'application/json'
             },
             timeout: 10 * 1000
         });
+
+        // @ts-ignore
+        axiosRetry(axiosClient, {
+            retries: 10,
+            retryDelay: (retryCount) => {
+                this.logger.info(`Retry delay ${retryCount * 1000} s`);
```

Comment:
Add a `const delay = retryCount * 1000;` for reuse here and in return
```suggestion
                this.logger.info(`Retry delay ${delay} s`);
```